tools/xnvme.c

```diff
@@ -1551,6 +1599,36 @@ static struct xnvme_cli_sub g_subs[] = {
 			XNVME_CLI_ADMIN_OPTS,
 		},
 	},
+	{
+		"subsystem-reset",
+		"Resets the subsystem",
+		"Resets the subsystem",
+		subsystem_reset,
+		{
+			{XNVME_CLI_OPT_POSA_TITLE, XNVME_CLI_SKIP},
+			{XNVME_CLI_OPT_URI, XNVME_CLI_POSA},
```

Comment:
For each of these, we should add `XNVME_CLI_ADMIN_OPTS` to be able to specify `be` etc.